File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        i=0
        j=len(numbers)-1
        while(i<j):
            currsum=numbers[i]+numbers[j]
            if(currsum==target):
                return[i+1,j+1]
            elif(currsum<target):
                i+=1
            else:
                j-=1
        return []

File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int i=0;
        int j=numbers.length-1;
        while(i<j)
        {
            int currsum=numbers[i]+numbers[j];
            if (currsum==target)
            {
                return new int[]{i+1,j+1};
            }
            else if (currsum<target)
            {
                i++;
            }
            else 
            {
                j--;
            }

        }return  new int[]{};
    } 
}

